### DISCUSSION TITLE
ISO install: Error setting up software source.

### DISCUSSION THREAD

**Discussion by h4m5t** [Security-Onion-Solutions/securityonion#10589] [2.4]
When I install 2.4.2-20230531 ISO image  with VMware, It says : Error setting up software source.
I Selected from CD/DVD ,local ISO files , Network. But they all failed.

**Comment by dougburks** [comment]
Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/VERIFY_ISO.md?

**Reply by h4m5t** [reply]
The hash value I calculated is different from the one in the document.I don't know why

PS E:\securityonion2.4> Get-FileHash .\securityonion-2.4.2-20230531.iso -Algorithm SHA256

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          62E481A416A83573E5A51623CB011B20BE7541C3557DDD47A2107B9D824FAB06

**Reply by dougburks** [reply]
Most likely this means that your download was interrupted or corrupted. Please try downloading again. If it still fails to verify, try downloading from another computer or another network.

**Comment by h4m5t** [comment]
Thx.
